src/components/collection/Config.tsx

```diff
@@ -69,23 +70,25 @@ function CollectionConfig({
                 </>
             }
         >
-            {resourceConfigHydrationErrorsExist ? (
-                <AlertBox
-                    severity="error"
-                    title={
-                        <FormattedMessage id="workflows.error.initFormSection" />
-                    }
-                    short
-                >
-                    <MessageWithLink messageID="error.message" />
-                </AlertBox>
-            ) : (
-                <BindingsMultiEditor
-                    draftSpecs={draftSpecs}
-                    readOnly={readOnly}
-                    RediscoverButton={RediscoverButton}
-                />
-            )}
+            <ErrorBoundryWrapper>
```

Comment:
Added an error boundry lower in the stack so that if this happens the section header still displays and wraps the error for a better ux